chromadb/db/impl/sqlite_pool.py

```diff
@@ -18,11 +18,11 @@ def __init__(
         self._pool = pool
         self._db_file = db_file
         self._conn = sqlite3.connect(
-            db_file, timeout=1000, check_same_thread=False, *args, **kwargs
-        )
+            db_file, timeout=1000, check_same_thread=False, uri=is_uri, *args, **kwargs
+        )  # type: ignore
```

Comment:
to silence a few of these mypy errors:
`"connect" gets multiple values for keyword argument "timeout"  [misc]mypy(error)`